http/src/main/resources/init-resources/startup.sh

```diff
@@ -77,27 +106,23 @@ if [ "$UPDATE_WELDER" == "true" ] ; then
     gcloud auth configure-docker
     docker-compose -f /etc/welder-docker-compose.yaml stop
     docker-compose -f /etc/welder-docker-compose.yaml rm -f
-    docker-compose -f /etc/welder-docker-compose.yaml up -d
+    docker-compose -f /etc/welder-docker-compose.yaml up -d &> start_output.txt || EXIT_CODE=$?
```

Comment:
Same question as above.. not sure if we want to reuse `start_output.txt` which is intended for user-specified start scripts